openlibrary/plugins/upstream/mybooks.py

```diff
@@ -342,8 +343,20 @@ def GET(self, username, key='want-to-read'):
                 }
                 for w in books
             ]
+
+            if len(records_json) < limit:
+                num_found = len(records_json)
+            else:
+                num_found = readlog.count_shelf(key)
```

Comment:
I realize there is likely an obvious explanation for this, but why only show the number of results on that page of it's the last page, and the total count every where else? At least that is how I am reading this.